src/04-modality-specific-files/08-genetic-descriptor.md

```diff
@@ -0,0 +1,116 @@
+sub-patient01	33	F	patient	489634	no
+```
+
+## Genetic Information
+
+Template:
+
+```Text
+genetic_info.json
+```
+
+The `genetic_info.json` file describes the genetic information available in the
+`participants.tsv` file and/or the genetic database described in
+`dataset_description.json`.
+Datasets containing the `Genetics` field in `dataset_description.json` or the
+`genetic_id` column in `participants.tsv` MUST include this file with the following
+fields:
+
+| Field name         | Definition                                                                  | Values                                                                                                                                                                                                             |
+| :----------------- | :-------------------------------------------------------------------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| GeneticLevel       | REQUIRED. Describes the level of analysis                                   | `Genetic`, `Genomic`, `Epigenomic`, `Transcriptomic`, `Metabolomic`, or `Proteomic`                                                                                                                                |
+| AnalyticalApproach | OPTIONAL Methodology or methodologies used to analyse the GeneticLevel      | String or list of strings. Each Value must be taken from the [database of Genotypes and Phenotypes (dbGaP)][gapsolr] under /Study/Molecular Data Type, for instance `SNP Genotypes (Array)` or `Methylation (CpG)` |
+| SampleOrigin       | REQUIRED. Describes from which tissue the genetic information was extracted | `blood`, `saliva`, `brain`, `csf`, `breast milk`, `bile`, `amniotic fluid`, `other biospecimen`                                                                                                                    |
+| TissueOrigin       | OPTIONAL Describes the type of tissue analyzed for SampleOrigin `brain`     | `gray matter`, `white matter`, `csf`, `meninges`, `macrovascular` or `microvascular`                                                                                                                               |
+| BrainLocation      | OPTIONAL Refers to the location in space of the TissueOrigin                | `MNI coordinate` or a `label` taken from the [Allen Brain Atlas][allen] possibly `layer` to refer to layer-specific gene expression, which can also tie up with laminar fMRI                                       |
+| CellType           | OPTIONAL Describes the type of cell analyzed                                | Value should come from the [cell ontology][ontology]                                                                                                                                                               |
```

Comment:
Adding periods (`.`) after OPTIONAL/REQUIRED. Spaces removed to retain table alignment.

```suggestion
| AnalyticalApproach | OPTIONAL. Methodology or methodologies used to analyse the GeneticLevel     | String or list of strings. Each Value must be taken from the [database of Genotypes and Phenotypes (dbGaP)][gapsolr] under /Study/Molecular Data Type, for instance `SNP Genotypes (Array)` or `Methylation (CpG)` |
| SampleOrigin       | REQUIRED. Describes from which tissue the genetic information was extracted | `blood`, `saliva`, `brain`, `csf`, `breast milk`, `bile`, `amniotic fluid`, `other biospecimen`                                                                                                                    |
| TissueOrigin       | OPTIONAL. Describes the type of tissue analyzed for SampleOrigin `brain`    | `gray matter`, `white matter`, `csf`, `meninges`, `macrovascular` or `microvascular`                                                                                                                               |
| BrainLocation      | OPTIONAL. Refers to the location in space of the TissueOrigin               | `MNI coordinate` or a `label` taken from the [Allen Brain Atlas][allen] possibly `layer` to refer to layer-specific gene expression, which can also tie up with laminar fMRI                                       |
| CellType           | OPTIONAL. Describes the type of cell analyzed                               | Value should come from the [cell ontology][ontology]                                                                                                                                                               |
```